Sources/StreamChat/Workers/ChannelMemberListUpdater.swift

```diff
@@ -11,32 +11,59 @@ class ChannelMemberListUpdater: Worker {
     ///   - query: The query used in the request.
     ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
     func load(_ query: ChannelMemberListQuery, completion: ((Error?) -> Void)? = nil) {
+        load(query, includeMembersResponse: false, completion: { result in
+            completion?(result.error)
+        })
+    }
+    
+    private func load(_ query: ChannelMemberListQuery, includeMembersResponse: Bool, completion: ((Result<[ChatChannelMember], Error>) -> Void)? = nil) {
```

Comment:
The current load method only returned error, I wanted to return the update ChatChannelMember so that I can use it for MemberState. Therefore, I added a private method with a boolean for not requiring existing code to initialise ChatChannelMember when it does not need it.

Reply:
@laevandus I don't it is necessary to maintain both functions. From now one, we should always prefer to return the responses 👍

Reply:
Yes, I will change this like I did in the other PR where you suggested it. Thanks!